assets/activation-scripts/activate

```diff
@@ -108,10 +121,7 @@ fi
 #       container invocations, and it would have the incorrect value for
 #       nested flox activations.
 _FLOX_ENV="$($_coreutils/bin/dirname -- "${BASH_SOURCE[0]}")"
-if [ -n "${FLOX_ENV:-}" ] && [ "$FLOX_ENV" != "$_FLOX_ENV" ]; then
-  echo "WARN: detected change in FLOX_ENV: $FLOX_ENV -> $_FLOX_ENV" >&2
```

Comment:
nonblocking question: when would we hit this?

Reply:
I tried looking back in the history and couldn't find a comment explaining an answer to that question. It also appears to be something that I myself added, so I _think_ I was just looking to detect and understand any cases where this could happen. At this point I'm happy dropping the logic.